src/components/blog/BlogHeader.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export default function BlogHeader(props) {
+  const latestPosts = props.allPosts[0];
+  const latestPostImage = props.postImage;
+
+  console.log(latestPostImage);
+
+  return (
+    <>
+      {latestPostImage && (
+        <div className="blogHeader">
+          <div className="headerImgBox">
+            <img
+              className="headerImg"
+              src={latestPostImage.source_url}
```

Comment:
eg source_url is not camelcase, is this because you're getting it from the wordpress api?

Reply:
Yes ,we are getting this from wordpress api